engine/client/drivers/docker.go

```diff
@@ -120,12 +120,12 @@ func (d *dockerDriver) create(ctx context.Context, imageRef string, containerNam
 	}
 
 	// ensure the image is pulled
-	if _, err := traceExec(ctx, exec.CommandContext(ctx, "docker", "inspect", "--type=image", imageRef), telemetry.Encapsulated()); err != nil {
+	if _, err := traceExec(ctx, exec.CommandContext(ctx, "docker", "inspect", "--type=image", imageRef), true, telemetry.Encapsulated()); err != nil {
```

Comment:
this SGTM initially. The only concern that I have is that if this `docker inspect` fails for some other reason besides not finding the images (docker engine connection error), then the trace will be marked as non-error incorrectly. 

Having said that, I understand it should be rare given that `collectLeftOverEngines` is called above which should error first